Indexing-Contract-Data-TheGraph.md

```diff
@@ -375,7 +375,7 @@ Now go to `styles` folder and replace all the contents of `Home.modules.css` fil
 }
 ```
 
-Lets now write some code to query the graph, create a new folder inside your `my-app` folder and name it `queries`. Inside the `queries` folder create a new file named `index.js` and paste the following lines of code:
+Let's now write some code to query the graph, create a new folder inside your `my-app` folder and name it `queries`. Inside the `queries` folder create a new file named `index.js` and paste the following lines of code:
```

Comment:
Could you change this line into -
`Let's write some code....`. 

So, basically removing the word `now`.